rtgui/main.cc

```diff
@@ -321,15 +309,9 @@ int main(int argc, char **argv)
         Gtk::Settings::get_for_screen(screen)->property_gtk_theme_name() = "Adwaita";
         Gtk::Settings::get_for_screen(screen)->property_gtk_application_prefer_dark_theme() = true;
 
-        Glib::ustring filename = argv0 + "/themes/RawTherapee.css";
+        Glib::ustring filename = Glib::build_filename(argv0 + "/themes/", options.theme + ".css");
```

Comment:
Couldn't this be replaced by the simpler `Glib::build_filename(argv0, "themes", options.theme + ".css")`?